chcemvediet/apps/accounts/tests/test_forms.py

```diff
@@ -1,13 +1,13 @@
 # vim: expandtab
 # -*- coding: utf-8 -*-
-import unittest
-
-from django.core.urlresolvers import reverse
 from django.contrib.auth.models import User
 from django.test import TestCase
 
+from poleno.utils.urls import reverse
```

Comment:
@martinmacko47 Vo viacerych suborov bude treba zmenit import na `from poleno.utils.urls import reverse`

Reply:
@viliambalaz Kludne to mozes zmenit naraz vsade v separatnom PR.

Reply:
https://github.com/martinmacko47/chcemvediet/issues/280#issuecomment-697155685